src/raster/rasterband.rs

```diff
@@ -600,3 +612,51 @@ impl ColorInterpretation {
         _string(rv)
     }
 }
+
+pub struct ColorTable<'a> {
+    c_color_table: GDALColorTableH,
+    raster_band: &'a RasterBand<'a>,
+}
+
+impl<'a> ColorTable<'a> {
+    pub fn raster_band(&self) -> &RasterBand {
+        self.raster_band
+    }
```

Comment:
Why do we need to return the raster band?

Reply:
Okay, I vote for removing it.

Reply:
It's removed.